bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -950,8 +950,8 @@ async def about_command(self, ctx: Context) -> None:
                 "48 hours. The staff then selected the best features from all the best teams, and made modifications "
                 "to ensure they would all work together before integrating them into the community bot.\n\n"
                 "It was a tight race, but in the end, <@!104749643715387392> and <@!303940835005825024> "
-                "walked away as grand champions. Make sure you check out `!snakes sal`, `!snakes draw` "
-                "and `!snakes hatch` to see what they came up with."
+                f"walked away as grand champions. Make sure you check out `{ctx.prefix}snakes sal`, `{ctx.prefix}snakes draw` "
```

Comment:
This line is now too long - it won't pass our linter. Please adjust so that it fits into 120 characters (only need to move the "draw" word on the next line).

The reason why the checks didn't trigger here is due to the conflict with `master`.

Reply:
You can also run the lint locally using `pipenv run lint`.